modules/prebid/ortb2blocking/analytics.go

```diff
@@ -0,0 +1,94 @@
+package ortb2blocking
+
+import (
+	"github.com/prebid/prebid-server/hooks/hookanalytics"
+	"github.com/prebid/prebid-server/hooks/hookstage"
+)
+
+const enforceBlockingTag = "enforce_blocking"
+
+const (
+	attributesAnalyticKey = "attributes"
+	badvAnalyticKey       = "adomain"
+	cattaxAnalyticKey     = "bcat"
```

Comment:
Is "cattax" the same as "bcat" Should it be two separate constants?

Reply:
I also got confused about this, but that's how it was done in PBS-Java: https://github.com/prebid/prebid-server-java/blob/master/extra/modules/ortb2-blocking/src/main/java/org/prebid/server/hooks/modules/ortb2/blocking/core/BidsBlocker.java#L284

Reply:
There is a constant `CATTAX_ATTRIBUTE = "cattax";`, weird, is this a mistake?

Reply:
For the `attributes` field (which is a list of all failed attributes) inside the `analytics` object PBS-JAVA uses the `CATTAX_ATTRIBUTE`, so we can see the cattax attribute as a separate one (f.i. `badv, bcat, cattax, bapp`)
but when listing the concrete values for the failed attributes inside the `analytics` object, it seems that PBS-Java has given a priority for `cattax` over `bcat`. So if both of them fail (bcat with category 'badCategory' and cattax with '2') we will see only the cattax listed under the bcat key in the analytics object and I have this covered in the last test.
Maybe thats a bug in PBS-Java but on the other hand the cattax was added in a later PR and it seemed that they've given the priority to it on purpose.

Reply:
I see, thank you for the explanation!